meta-balena-common/recipes-containers/mobynit/mobynit_1.0.bb

```diff
@@ -0,0 +1,32 @@
+HOMEPAGE = "https://www.balena.io/"
```

Comment:
not sure how to do it for yocto but we should add the equivalent of https://devmanual.gentoo.org/eclass-reference/ebuild/#lbAF
```
RESTRICT="strip"
```

to prevent the binaries from being stripped by yocto... rather the go build should do
`go build -ldflags "-s -w" ...` (https://golang.org/cmd/link/)

Reply:
ack, I'll look into this.